web_console/models/federation.js

```diff
@@ -52,6 +52,12 @@ module.exports = (sequelize, DataTypes) => {
       default: null,
       comment: 'used for authorization. null stands for a passive pair, others stands for initiative pair',
     },
+    k8s_settings: {
+      type: DataTypes.TEXT('long'),
+      allowNull: true,
+      default: null,
+      comment: 'settings for kubernetes cluster',
+    },
```

Comment:
if `k8s_settings` must be JSON-format, `getterMethods` and `setterMethods` could be defined for it, just like what `ticket.public_params` do. so that we can read & write `k8s_settings` as an object over runtime, and make JSON encoding/decoding automatically